packages/unity-react-core/src/components/ImageBasedCard/ImageBasedCard.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import { GaEventWrapper } from "../GaEventWrapper/GaEventWrapper";
+
+export type CardOrientation = "portrait" | "landscape" | "square";
+export type CardSize = "sm" | "md" | "lg";
+
+export interface ImageBasedCardProps {
+  orientation: CardOrientation;
+  size: CardSize;
+  image: string;
+  title: string;
+  buttonText: string;
+  buttonHref?: string;
+  gaSection?: string;
+  gaButtonText?: string;
```

Comment:
Looks like we're using `buttonText` but not `gaButtonText`. Does `gaButtonText` need to be in the interface?